renderer/src/pages/Dashboard.tsx

```diff
@@ -1,55 +1,33 @@
 import { useEffect, useState } from 'react'
-import {
-  getAllActivities, stopSaturnNode,
-  setFilAddress, getFilAddress,
-  getTotalEarnings, getTotalJobsCompleted
-} from '../lib/station-config'
-import { ActivityEventMessage } from '../typings'
+import { stopSaturnNode, setDestinationWalletAddress, getDestinationWalletAddress } from '../lib/station-config'
 import ActivityLog from '../components/ActivityLog'
 import HeaderBackgroundImage from '../assets/img/header.png'
 import WalletIcon from '../assets/img/wallet.svg'
 import { useNavigate } from 'react-router-dom'
 import { confirmChangeWalletAddress } from '../lib/dialogs'
 import UpdateBanner from '../components/UpdateBanner'
+import useStationActivity from '../hooks/StationActivity'
 
 const Dashboard = (): JSX.Element => {
   const navigate = useNavigate()
-
   const [address, setAddress] = useState<string | undefined>()
-  const [totalJobs, setTotalJobs] = useState<number>(0)
-  const [totalEarnings, setTotalEarnigs] = useState<number>(0)
-  const [activities, setActivities] = useState<ActivityEventMessage[]>([])
+  const { totalJobs, totalEarnings, activities } = useStationActivity()
   const shortAddress = (str: string) => str
     ? str.substring(0, 4) + '...' + str.substring(str.length - 4, str.length)
     : ''
   const disconnect = async () => {
```

Comment:
this isn't a disconnect any more, and the saturn node shouldn't be stopped

Reply:
This is addressed on #334